libraries/botbuilder-core/src/testAdapter.ts

```diff
@@ -391,6 +395,51 @@ export class TestAdapter extends BotAdapter implements IUserTokenProvider {
         return undefined;
     }
 
+    
+    private _exchangeableTokens : {[key: string]: ExchangeableToken} = {};
```

Comment:
blank line after private member

Reply:
Also private members in js don't require to start with `_`, so I'd rename this private member to `exchangeableTokens`